pkg/cache/cache.go

```diff
@@ -61,7 +61,8 @@ func New(client client.Client) *Cache {
 	}
 }
 
-type Resources map[corev1.ResourceName]map[string]int64
+type Resources map[corev1.ResourceName]FlavorResources
+type FlavorResources map[string]int64
```

Comment:
We have ResourceFlavor API, this type name is going to be confusing. 

I suggest we keep it as is, and just merge the changes to `cache_test.go`

Reply:
This is also what I'm hesitate about, I'll follow you advices then.

Reply:
updated.